frontend/src/components/pages/RegistrantExperience/index.tsx

```diff
@@ -125,15 +140,23 @@ const RegistrantExperiencePage = (): React.ReactElement => {
         isReviewRegistrationFilled={isReviewRegistrationFilled}
         setCurrentStep={setCurrentStep}
       />
-      <Box mt="144px" mb="168px" mx="10vw">
-        {getCurrentRegistrantStepComponent(currentStep)}
-      </Box>
+      {!isLoading && (
+        <Box mx="10vw">
+          {camp ? (
+            getCurrentRegistrantStepComponent(currentStep)
+          ) : (
+            <Text>Error: Camp not found. Please go back and try again.</Text>
+          )}
+        </Box>
+      )}
+
+      {isLoading && <Spinner justifySelf="center" />}
```

Comment:
I don't think the spinner is centred on the page, but I think we can just make this a backlog ticket. The request is pretty quick anyways